elyra/pipeline/pipeline_constants.py

```diff
@@ -17,3 +17,4 @@
 PIPELINE_DEFAULTS = "pipeline_defaults"
 RUNTIME_IMAGE = "runtime_image"
 ENV_VARIABLES = "env_vars"
+PIPELINE_META_PROPERTIES = ["name", "description", "runtime", PIPELINE_DEFAULTS]
```

Comment:
I think I'll remove `PIPELINE_DEFAULTS` from this list since it technically isn't a "meta-property", and make the corresponding change in `pipeline_definition.py:175`